### PR TITLE
Split set.pass test

### PR DESCRIPTION
**Pull request by SergeyKopienko**
In this PR we split `set.pass` tests into some small separate tests due rime-out error during tests runs.

### CODE REVIEW

**Comment by dmitriy-sobolev** [comment]
Let's scatter these cases into `set_union`, `set_intersection`, `set_difference`, `set_symmetric_difference` tests. Splitting the tests based on checked algorithms looks more justified because this is what is done in majority of oneDPL tests and the issues are more likely to group for a specific algorithm rather than a type.

**Comment by dmitriy-sobolev** [comment]
Do you split the test because it too big and results in timeouts?

**Comment by SergeyKopienko** [comment]
> Let's scatter these cases into `set_union`, `set_intersection`, `set_difference`, `set_symmetric_difference` tests. Splitting the tests based on checked algorithms looks more justified because this is what is done in majority of oneDPL tests and the issues are more likely to group for a specific algorithm rather than a type.

Done.

File: test/parallel_api/algorithm/alg.sorting/alg.set.operations/set_test_symmetric_difference.pass.cpp
Comment (SergeyKopienko) [reply test/parallel_api/algorithm/alg.sorting/alg.set.operations/set_test_symmetric_difference.pass.cpp]: Fixed

File: test/parallel_api/algorithm/alg.sorting/alg.set.operations/set_test_symmetric_difference.pass.cpp
Comment (SergeyKopienko) [reply test/parallel_api/algorithm/alg.sorting/alg.set.operations/set_test_symmetric_difference.pass.cpp]: Fixed

**Review by dmitriy-sobolev — APPROVED** [review]
LGTM. As usual, please wait until CI successfully passes before merging.

**Pull request merged**
